### PR TITLE
Fixes #21291: systemUpdate technique parameters should accept properties as input

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/21291

### CODE REVIEW

**Comment by Fdall** [comment]
Do not merge yet

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
PR updated with a new commit

File: techniques/applications/systemUpdate/2.0/metadata.xml [hunk 1]
```diff
@@ -37,34 +37,18 @@ The technique will automatically run a Rudder inventory after the update if it w
       </SECTION>
     </SECTION>
     <SECTION name="Campaign Infos" multivalued="false" component="false">
-      <SELECT1>
+      <INPUT>
         <NAME>ENABLE_UPDATE</NAME>
-        <DESCRIPTION>Activate or not the update</DESCRIPTION>
```
Comment (VinceMacBuche) [inline techniques/applications/systemUpdate/2.0/metadata.xml]: you should explain that you can use properties, especially group properties

**Comment by amousset** [comment]
How will it work for existing directives?

**Comment by VinceMacBuche** [comment]
What the behavior when you update from 1.0 to 2.0 ?

**Review by VinceMacBuche — APPROVED** [review]

**Comment by Fdall** [comment]
PR updated with a new commit

**Pull request merged**